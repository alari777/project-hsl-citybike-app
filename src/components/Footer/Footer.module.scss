.footer {
  background-image: linear-gradient(
                  to bottom,
                  rgba(0, 0, 0, 0.7),
                  rgba(0, 0, 0, 0.5)
  );
  text-align: center;
  font-size: 0.75rem;
  display: flex;
  justify-content: center;
  align-items: center;
  text-transform: uppercase;
  min-height: 5rem;
}
