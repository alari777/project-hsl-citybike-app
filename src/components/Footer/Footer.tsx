import { FC } from 'react';

const Footer: FC = () => {
  return <footer>Helsinki city bike. Created at NextJS. 2023</footer>;
};

export default Footer;
