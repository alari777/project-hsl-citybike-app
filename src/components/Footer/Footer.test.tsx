import { render, screen } from '@testing-library/react';
import Footer from './Footer';

it('should render footer', () => {
  render(<Footer />);
  const footer = screen.getByText(
    /Helsinki city bike. Created at NextJS. 2023/i
  );
  expect(footer).toBeInTheDocument();
  expect(footer).toMatchSnapshot();
});
