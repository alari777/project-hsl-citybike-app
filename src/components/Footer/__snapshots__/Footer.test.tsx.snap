// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer 1`] = `
<footer
  class="footer"
>
  Helsinki city bike. Created at NextJS. 2023
</footer>
`;
